homeassistant/components/nws/weather.py

```diff
@@ -141,6 +141,8 @@ def __init__(
         latitude = entry_data[CONF_LATITUDE]
         longitude = entry_data[CONF_LONGITUDE]
         self.coordinator_forecast_legacy = nws_data.coordinator_forecast
+        self.coordinator_forecast_twice_daily = nws_data.coordinator_forecast
```

Comment:
We don't need these, they are already available as `self.forecast_coordinators["twice_daily"]` and similar the hourly.